signal.go

```diff
@@ -71,7 +58,7 @@ func KillAll(ctx context.Context, procs []Process) error {
 	}
 
 	for _, proc := range procs {
-		proc.Wait(ctx)
+		_, _ = proc.Wait(ctx)
```

Comment:
should we catch this error.

Reply:
Since this is `SIGKILL`, `Wait()` will always return an error when the process is killed.